scripts/generate-specs.sh

```diff
@@ -53,33 +53,30 @@ main() {
   TEMP_OUTPUT_DIR="$TEMP_DIR/out"
   SCHEMA_FILE="$TEMP_DIR/schema.json"
 
-  CODEGEN_REPO_PATH="$RN_DIR/packages/react-native-codegen"
-  CODEGEN_NPM_PATH="$RN_DIR/../react-native-codegen"
-
-if [ -z "$NODE_BINARY" ]; then
+  if [ -z "$NODE_BINARY" ]; then
     echo "Error: Could not find node. Make sure it is in bash PATH or set the NODE_BINARY environment variable." 1>&2
     exit 1
   fi
 
-  if [ -d "$CODEGEN_REPO_PATH" ]; then
-    CODEGEN_PATH=$(cd "$CODEGEN_REPO_PATH" && pwd)
-  elif [ -d "$CODEGEN_NPM_PATH" ]; then
-    CODEGEN_PATH=$(cd "$CODEGEN_NPM_PATH" && pwd)
-  else
-    echo "Error: Could not determine react-native-codegen location. Try running 'yarn install' or 'npm install' in your project root." 1>&2
+  # Check if user-defined CODEGEN_PATH is correct
+  if [[ "$CODEGEN_PATH" && ! -d "$CODEGEN_PATH" ]]; then
```

Comment:
One thing that I just realised is that you won't be able to set this unless the Script Phase is part of a user project. Right now, it's inside `FBReactNativeSpec` target.

So, I will most likely drop it.